customer_outstanding_statement/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Print Customer Outstanding Statement
+====================================
+
+The outstanding statement provides details of all outstanding customer receivables
+up to a particular date. This includes all unpaid invoices, unclaimed refunds and
+outstanding payments. The list is displayed in chronological order and is split by currencies.
+
+Aging details can be shown in the report, expressed in aging buckets (30 days
+due, ...), so the customer can review how much is open, due or overdue.
+
+Configuration
+=============
+
+Users willing to access to this report should have proper Accounting & Finance rights:
+
+#. Go to *Settings / Users* and edit your user to add the corresponding access rights as follows.
+#. In *Application / Accounting & Finance*, select *Billing* or *Billing Manager*
+#. In *Technical Setting* mark *Show Full Accounting Features* options.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Invoicing > Sales > Master Data > Customers and select one or more
```

Comment:
Sales or Purchases. Put the two or neither. :)

Reply:
@mreficent The report only takes receivable account, and you can only have receivables in sales, basically

Reply:
True 😓